modules/caddyhttp/matchers.go

```diff
@@ -1469,6 +1475,13 @@ func (mre *MatchRegexp) Match(input string, repl *caddy.Replacer) bool {
 // UnmarshalCaddyfile implements caddyfile.Unmarshaler.
 func (mre *MatchRegexp) UnmarshalCaddyfile(d *caddyfile.Dispenser) error {
 	for d.Next() {
+		// If this is the second iteration of the loop
+		// then there's more than one path_regexp matcher
+		// and we would end up overwriting the old one
+		if mre.Pattern != "" {
+			return d.Err("path_regexp matcher can only be used once per named matcher")
```

Comment:
```suggestion
			return d.Err("regular expression can only be used once per named matcher")
```

This is used for more than just the path matcher IIRC

Reply:
It's not used for header_regexp so I don't know what else 🤔

Reply:
True, but this is an embeddable type, it just so happens that path is the only one that does it for right now.